ruby/vampires.rb

```diff
@@ -0,0 +1,47 @@
+puts "How many employees will be processed?"
+employee_number = gets.chomp
+n = employee_number.to_i
+n.times do
+  puts "What's your name?"
+  employee_name = gets.chomp
+  puts "How old are you? What year were you born? (right/ wrong)"
+  age = gets.chomp
+  puts "Our company cafeteria serves garlic bread. Should we worder some for you (y/n)?"
+  garlic_bread = gets.chomp
+  puts "Would you like to enroll in the company's health insurance (y/n)?"
+  insurance = gets.chomp
+
+  if employee_name == "Drake Cula" || employee_name == "Tu Fang"
+    result = "Definitely a vampire."
+
+    elsif age == "right" && (garlic_bread == "y" || insurance == "y")
```

Comment:
We are repeating ourselves a bit with things like  `garlic_bread == 'y'` so what we could do is set a variable like
```likes_garlic_bread = garlic_bread == 'y'``` and now we can use that variable for the comparison.  If we have to check if they don't like garlic specifically we can then do `!likes_garlic_bread`.